### PR TITLE
Upgrade http-interop

### PR DESCRIPTION
**Pull request by andig**
Refs https://github.com/php-pm/php-pm/pull/335

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "php-pm/httpkernel-adapter",
     "license": "MIT",
     "require": {
-        "php-pm/php-pm": "^1.0",
+        "php-pm/php-pm": "^1.1",
```
Comment (andreybolonin) [inline composer.json]: maybe `"php-pm/php-pm": "^1.1|dev-master",`  ?
screen shot 2018-03-22 at 2 24 59 pm

**Comment by andreybolonin** [comment]
https://github.com/php-pm/php-pm-httpkernel/issues/97

**Comment by andig** [comment]
@andreybolonin could you please stop spamming PRs with the same off-topic issue?

File: composer.json (hunk 1 above)
Comment (andig) [reply composer.json]: Never, don't think so. Use branch aliases if you need dev-master.

**Pull request merged**